app/src/redux/eventReducers.js

```diff
@@ -1,13 +1,15 @@
+import { API } from './../helper/constants';
```

Comment:
It will not be working, while Ura's PR isn't merged.
I will correct it, when it will be done.